### PR TITLE
[Merged by Bors] - feat(LinearAlgebra/Quotient): `(∀ a :  M ⧸ p, P a) ↔ ∀ a : M, P ⟦a⟧`

### PR DESCRIPTION
**Pull request by YaelDillies**
From PFR

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/LinearAlgebra/Quotient.lean [hunk 1]
```diff
@@ -104,6 +104,8 @@ theorem mk_neg : (mk (-x) : M ⧸ p) = -(mk x : M ⧸ p) :=
 theorem mk_sub : (mk (x - y) : M ⧸ p) = (mk x : M ⧸ p) - (mk y : M ⧸ p) :=
   rfl
 
+nonrec lemma «forall» {P : M ⧸ p → Prop} : (∀ a, P a) ↔ ∀ a : M, P ⟦a⟧ := Quotient.forall
```
Comment (Ruben-VandeVelde) [inline Mathlib/LinearAlgebra/Quotient.lean:107]: Isn't `mk a` going to work better?

File: Mathlib/LinearAlgebra/Quotient.lean [hunk 2]
```diff
@@ -104,6 +104,8 @@ theorem mk_neg : (mk (-x) : M ⧸ p) = -(mk x : M ⧸ p) :=
 theorem mk_sub : (mk (x - y) : M ⧸ p) = (mk x : M ⧸ p) - (mk y : M ⧸ p) :=
   rfl
 
+nonrec lemma «forall» {P : M ⧸ p → Prop} : (∀ a, P a) ↔ ∀ a, P (mk a) := Quotient.forall
```
Comment (jcommelin) [inline Mathlib/LinearAlgebra/Quotient.lean:107]: I think this should also be `protected`, but I don't know if that should go before or after `nonrec`.
```suggestion
nonrec lemma «forall» {P : M ⧸ p → Prop} : (∀ a, P a) ↔ ∀ a, P (mk a) := Quotient.forall
```

**Review by jcommelin — COMMENTED** [review]
bors d+

File: Mathlib/LinearAlgebra/Quotient.lean (hunk 2 above)
Comment (YaelDillies) [reply Mathlib/LinearAlgebra/Quotient.lean]: TIL it goes before

**Pull request closed without merging**